servicebroker/test/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
When we do cmd "mocha init <folder>", then the index.html and others files are automatically added by mocha.